docs/rules/prefer-node-protocol.md

```diff
@@ -11,6 +11,12 @@ When importing builtin modules, it's better to use the [`node:` protocol](https:
 
 And don't forget to [upvote this issue](https://github.com/nodejs/node/issues/38343) if you agree.
 
+Note that this feature is not available in early versions of Node 12 and Node 14:
```

Comment:
I don't think we need to mention Node.js 12 as it's not supported anymore.

Reply:
I removed "Node 12" from this sentence. However, I do think we should continue to mention it in the versions list below for completeness / to accurately satisfy the wording "Node.js support for this feature began in". Once all of eslint-plugin-unicorn's Node versions support this feature, I think we can just remove this section completely.